### PR TITLE
Doc: add the control profiling guide spec

### PR DESCRIPTION
**Pull request by macDure**
add the control profiling guide spec

### CODE REVIEW

**Comment by diegohu** [comment]
Please move fuel related docs to https://github.com/ApolloAuto/apollo/tree/master/docs/Apollo_Fuel

File: docs/specs/Control_Profiling/README.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+Here is the folder structure requirements for job submission:
+1. A cyber record file containing the execution of open space planner scenario.
+
+1. A configuration file `vehicle_param.pb.txt`; there is a sample file under `apollo/modules/common/data/vehicle_param.pb.txt`.
+
+### Submit job in Dreamland
+
+Go to [Apollo Dreamland](http://bce.apollo.auto/), login with **Baidu** account, choose `Apollo Fuel --> Jobs`，`New Job`, `Open Space Planner Profiling`，and input the correct BOS path as in [Upload data to BOS](###Upload-data-to-BOS) section：
+
```
Comment (kevin-y-wang) [inline docs/specs/Control_Profiling/README.md:64]: Typo: "Open Space Planner Profiling" should be "Control Profiling"

**Review by kevin-y-wang — CHANGES_REQUESTED** [review]

File: docs/specs/Control_Profiling/README.md (hunk 1 above)
Comment (macDure) [reply docs/specs/Control_Profiling/README.md]: fixed

**Comment by macDure** [comment]
> Please move fuel related docs to https://github.com/ApolloAuto/apollo/tree/master/docs/Apollo_Fuel

fixed

**Review by kevin-y-wang — APPROVED** [review]

**Pull request merged**